### ISSUE TITLE
How can I center the markers on the map?

### ISSUE THREAD

**Issue by memoxmrdl** [Matt-Jensen/ember-cli-g-maps#67]
I have a problem how can I auto-center the markers on the map?

**Comment by Matt-Jensen** [comment]
Well there's more than one way.  The most straightforward way I can think of is using the `center_changed` event of the g-maps component and applying the map center to the marker in question.  Would that work for you?

**Comment by memoxmrdl** [comment]
Hi Matt, yep I'm try use 'center_changed' but I have many centers that can't centering above on the map, I need something as following:

map.setCenter(new google.maps.LatLng(latitude, longitude));

**Comment by Matt-Jensen** [comment]
Just so I understand your issue correctly, you have a group of markers that need to be updated to the _new center of the map_ once the user has _changed the center of the map_?

I'm afraid I'm not sure what you mean by:

> I have many centers that can't centering above on the map

However the `map.setCenter(...)` code you have can be easily achieved with:
`set(this, 'myMapLat', newLat)` and `set(this, 'myMapLng', newLng)`

where:

`{{g-maps lat=myMapLat lng=myMapLng}}`

**Comment by memoxmrdl** [comment]
Yep! sorry for my English, I understand your solution let me try again. thanks
